db/subplugins.php

```diff
@@ -13,18 +13,22 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
-
 /**
  * Surveypro subplugin types declaration
  *
  * @package   mod_surveypro
  * @copyright 2013 onwards kordan <kordan@mclink.it>
  * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
-
 defined('MOODLE_INTERNAL') || die();
-
 $subplugins = array('surveyprofield' => 'mod/surveypro/field',
                     'surveyproformat' => 'mod/surveypro/format',
                     'surveyprotemplate' => 'mod/surveypro/template',
                     'surveyproreport' => 'mod/surveypro/report');
+
+// TODO: this file is here because of compatibility with versions earlier than 3.7.
```

Comment:
the change to json has been made in master that is 3.8. So this file remains here for < 3.8 moodle versions